HBW-181 Integrate new claiming components

vladdoncheg opened the pull request.

vladdoncheg commented:
![Скриншот 2019-10-16 10 50 17](https://user-images.githubusercontent.com/42524654/66902025-9911dc80-f008-11e9-9435-6c4a17822e13.png)

TimAle commented on app/views/layouts/_navigation.html.haml:
```diff
@@ -26,6 +26,10 @@
                 = t('.users')
+            %li
+              = link_to tasks_path do
+                %span.fas.fa-check-double
```
https://fontawesome.com/icons/tasks?style=solid

vladdoncheg commented:
- Removed rendering old button and list;
- Integrated new button and list;
- Fixed 'View all' path;
- Remade icon. Now it's a component, has right color and available to hover.


![Скриншот 2019-10-17 12 48 01](https://user-images.githubusercontent.com/42524654/67003132-047dac00-f0e6-11e9-91f7-e74c49dff5c5.png)

Hovering:
![Скриншот 2019-10-17 12 52 43](https://user-images.githubusercontent.com/42524654/67003133-05164280-f0e6-11e9-91a1-1316ae55596b.png)

Opened pop_up:
![Скриншот 2019-10-17 12 47 32](https://user-images.githubusercontent.com/42524654/67003131-047dac00-f0e6-11e9-9274-1bbc9882d0cc.png)

vladdoncheg commented:
Also:
- Simplyfied integrating button;
- Removed blinking 'No tasks' while fetching.
